internal/ee/rbac/policy.go

```diff
@@ -388,20 +408,26 @@ func (r *Rbac) EnforceAccount(
 	return nil
 }
 
-func setPolicy(e casbin.IEnforcer, policy []string) error {
+type setPolicyResult struct {
+	DidConflict bool
+}
+
+func setPolicy(e casbin.IEnforcer, policy []string) (*setPolicyResult, error) {
```

Comment:
the on conflict logic is kind of dead for now but leaving it in here as a final safety mechanism